src/test/java/com/google/graphgeckos/dashboard/reader/BuildBotSetUpReaderTest.java

```diff
@@ -0,0 +1,27 @@
+package com.google.graphgeckos.dashboard.reader;
```

Comment:
It would be great to have a test that verifies the behavior for a malformed input file, but you might skip this given the short remaining time.

Reply:
Using protos we wouldn't need that .